src/utils/summary.cpp

```diff
@@ -518,20 +534,27 @@ DataType lbann_summary::local_min(const Mat& mat) const {
   const El::Int ldim = mat.LDim();
   const DataType * __restrict__ mat_buf = mat.LockedBuffer();
   auto min = std::numeric_limits<DataType>::max();
+  int nthreads = omp_get_num_threads();
+  std::vector<EvalType> local_min(nthreads, std::numeric_limits<DataType>::max());
```

Comment:
This needs to be `DataType`, not `EvalType`, or it will fail to compile.

Reply:
Fixed.